News/Entities/UserEntity.cs

```diff
@@ -10,7 +10,7 @@ public class UserEntity
 
     public string? Login { get; set; }
 
-    public Guid? PasswordId { get; set; }
+    public Guid PasswordId { get; set; }
```

Comment:
Dont change it because we have google SSO and we dont have a Password in User. That a reason why here is Guid?
You should to change the approach in method GetByIdAsync.

Reply:
I returned "Guid?" for PasswordId. Without your comment, it looked like it was accidentally placed